### ISSUE TITLE
Entire atmosphere Total Cloud Cover [%]} variable from pgb files

### ISSUE THREAD

**Issue by wzzheng90** [NOAA-EMC/UPP#529]
I checked the variable "TCDCclm" {entire atmosphere (considered as a single layer) Total Cloud Cover [%]} from pgb files such as "gfs.t00z.pgrb2.0p25.f024".  It looks this is not Cloud Cover in %. The values are 0 to 450000.   However, "TCDCclm" from sfluxgrbf files (e.g., gfs.t00z.sfluxgrbf024.grib2) looks reasonable and its values are 0 to 100.

**Comment by WenMeng-NOAA** [comment]
@wzzheng90 Are you meaning TCDC from ops. gfs V16.2 or gfs V17? I don't see issue in gfs v16 on wcoss2:
```
wen.meng@clogin02 atmos$ pwd
/lfs/h1/ops/prod/com/gfs/v16.2/gfs.20220706/00/atmos
wen.meng@clogin02 atmos$ wgrib2 gfs.t00z.pgrb2.0p25.f024 -match "TCDC:entire atmosphere" -stats
636:455729662:ndata=1038240:undef=0:mean=66.3062:min=0:max=100:cos_wt_mean=61.8696
637:456554628:ndata=1038240:undef=0:mean=69.9018:min=0:max=100:cos_wt_mean=64.8935
```

**Comment by WenMeng-NOAA** [comment]
@wzzheng90 Which version of UPP and global-workflow are the UFS p8/p8c using? If global-workflow is from develop, you might consult with @WalterKolczynski-NOAA first. Some of gfs post processing scripts are moved from UPP repository to global-workflow repository. Also you might check master fiiles gfs*master* ( I don't see master files archived in your directories).

**Comment by wzzheng90** [comment]
@WenMeng-NOAA  It should be the latest version of global-workflow (P8):
commit 3076e13ca5976206f1cdab58cca1b6f80fbb7716 (HEAD -> develop, origin/develop, origin/HEAD)
Author: Fanglin Yang <fanglin.yang@noaa.gov>
Date:   Fri Jul 1 10:33:39 2022 -0400

**Comment by WalterKolczynski-NOAA** [comment]
Global workflow has been using the UFS version of UPP since late May. GW is currently using UFS tag Prototype-P8c, which is using UPP dc8bc68.

I don't know that any of the post-processing scripts we absorbed would break the variable in this way, but if I'm wrong I can certainly try to fix it. The variable doesn't appear in any of the scripts either.

**Comment by WenMeng-NOAA** [comment]
@WalterKolczynski-NOAA Thanks for clarifying. 
@wzzheng90 Do you have model history files and UPP master files archived?

**Comment by wzzheng90** [comment]
@WalterKolczynski-NOAA  Thanks!
@WenMeng-NOAA  I removed them.  However, if you want them, I can rerun for a short fcst time.

**Comment by wzzheng90** [comment]
@WenMeng-NOAA  @WalterKolczynski-NOAA   I recreated them.  Please see all output files: /scratch1/NCEPDEV/stmp2/Weizhong.Zheng/P8c_atmCTL/COMROOT/p8c2020012600/gfs.20200126/00/atmos

**Comment by wzzheng90** [comment]
@WenMeng-NOAA   Yes, I confirm that my experiment uses Thompson MP in P8/P8c (GFDL MP in P8b or previous versions).  So anything needs update in the UPP with Thompson MP?

**Comment by WenMeng-NOAA** [comment]
@wzzheng90 You might update the diag table for outputting 3D cloud fraction as "cldfra".
